src/fidesops/models/datasetconfig.py

```diff
@@ -148,9 +150,13 @@ def to_graph_field(field: FidesopsDatasetField) -> Field:
 
         (data_type_name, is_array) = parse_data_type_string(meta_section.data_type)
 
-    if field.fields:
-        sub_fields = [to_graph_field(fld) for fld in field.fields]
+        if meta_section.return_all_elements:
```

Comment:
This will set `return_all_elements` to `True` below, even if for some reason the value in the dataset has been manually specified as `return_all_elements: false`. I think the safest check would be `if meta_section.return_all_elements == "true": ...`

Reply:
It'd be great to get some tests in for this scenario too.

I think I see the nuance here, we don't want to default this field to `False` since it's only to be specified in array fields?